src/wallet/wallet.h

```diff
@@ -198,7 +199,7 @@ class CMerkleTx : public CTransaction
     int GetDepthInMainChain() const { const CBlockIndex *pindexRet; return GetDepthInMainChain(pindexRet); }
     bool IsInMainChain() const { const CBlockIndex *pindexRet; return GetDepthInMainChainINTERNAL(pindexRet) > 0; }
     int GetBlocksToMaturity() const;
-    bool AcceptToMemoryPool(bool fLimitFree=true, bool fRejectAbsurdFee=true);
+    bool AcceptToMemoryPool(bool fLimitFree=true, CFeeRate maxFeeRate=CFeeRate(::minRelayTxFee.GetFeePerK() * 10000));
```

Comment:
Can't we just get rid of the default value and require the caller to always pass it?